src/lang/extra/russian.txt

```diff
@@ -523,8 +523,8 @@ STR_CONFIG_SETTING_SIMULATE_SIGNALS                             :Симуляц
 STR_CONFIG_SETTING_SIMULATE_SIGNALS_HELPTEXT                    :Устанавливает целевое расстояние между светофорами для новых мостов и туннелей. Фактическое расстояние может немного отличаться от указанного, чтобы избежать неравномерного расположения. Изменение этого параметра не приводит к переоборудованию существующих мостов и туннелей со светофорами.
 STR_CONFIG_SETTING_SIMULATE_SIGNALS_VALUE                       :{COMMA} клет{P ку ки ок}
 
-STR_CONFIG_SETTING_DAY_LENGTH_FACTOR                            :Фактор продолжительности дня: {STRING}
-STR_CONFIG_SETTING_DAY_LENGTH_FACTOR_HELPTEXT                   :Темп игры замедляется из-за этого фактора.
+STR_CONFIG_SETTING_DAY_LENGTH_FACTOR                            :Коэффициент продолжительности дня: {STRING}
+STR_CONFIG_SETTING_DAY_LENGTH_FACTOR_HELPTEXT                   :Темп игры замедляется из-за этого коэффициента. При увеличении снижает нагрузку на процессор путём снижения количества расчётов изменений на карте.
```

Comment:
Can you please translate the sentence you just added into English, then append your English translation to the setting description in `english.txt`? https://github.com/JGRennison/OpenTTD-patches/blob/e57a7ad2733b74336a3869d0a421841a7046e8ac/src/lang/extra/english.txt#L573